src/contrib/serializers/Akka.Serialization.Hyperion/HyperionSerializer.cs

```diff
@@ -70,7 +70,8 @@ public HyperionSerializer(ExtendedActorSystem system, HyperionSerializerSettings
                     preserveObjectReferences: settings.PreserveObjectReferences,
                     versionTolerance: settings.VersionTolerance,
                     surrogates: new[] { akkaSurrogate },
-                    knownTypes: provider.GetKnownTypes()));
+                    knownTypes: provider.GetKnownTypes(), 
+                    ignoreISerializable:true));
```

Comment:
@Aaronontheweb But we don't have any alternatives for Exceptions. We can use only `ISerializable` serialization